### ISSUE TITLE
atomic-reactor fails on EL7 because of python-six version

### ISSUE THREAD

**Issue by maxamillion** [projectatomic/atomic-reactor#286]
```
[root@losbs sdodson-openshift-ansible]# atomic-reactor
Traceback (most recent call last):
  File "/usr/bin/atomic-reactor", line 9, in <module>
    load_entry_point('atomic-reactor==1.4.0', 'console_scripts', 'atomic-reactor')()
  File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 378, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 2566, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 2260, in load
    entry = __import__(self.module_name, globals(),globals(), ['__name__'])
  File "/usr/lib/python2.7/site-packages/atomic_reactor/cli/main.py", line 17, in <module>
    from atomic_reactor.api import build_image_here, build_image_in_privileged_container, build_image_using_hosts_docker
  File "/usr/lib/python2.7/site-packages/atomic_reactor/api.py", line 11, in <module>
    from atomic_reactor.inner import DockerBuildWorkflow
  File "/usr/lib/python2.7/site-packages/atomic_reactor/inner.py", line 16, in <module>
    from atomic_reactor.build import InsideBuilder
  File "/usr/lib/python2.7/site-packages/atomic_reactor/build.py", line 15, in <module>
    from atomic_reactor.core import DockerTasker, LastLogger
  File "/usr/lib/python2.7/site-packages/atomic_reactor/core.py", line 31, in <module>
    import docker
  File "/usr/lib/python2.7/site-packages/docker/__init__.py", line 20, in <module>
    from .client import Client, AutoVersionClient # flake8: noqa
  File "/usr/lib/python2.7/site-packages/docker/client.py", line 23, in <module>
    from . import clientbase
  File "/usr/lib/python2.7/site-packages/docker/clientbase.py", line 7, in <module>
    import websocket
  File "/usr/lib/python2.7/site-packages/websocket/__init__.py", line 22, in <module>
    from ._core import *
  File "/usr/lib/python2.7/site-packages/websocket/_core.py", line 41, in <module>
    from ._url import *
  File "/usr/lib/python2.7/site-packages/websocket/_url.py", line 23, in <module>
    from six.moves.urllib.parse import urlparse
ImportError: No module named urllib.parse
[root@losbs sdodson-openshift-ansible]# rpm -q python-six
python-six-1.3.0-4.el7.noarch
```

If I update the version of python-six from my [copr](https://copr.fedoraproject.org/coprs/maxamillion/epel7-python-six/) it works.
```
[root@losbs yum.repos.d]# yum update python-six
Loaded plugins: fastestmirror, langpacks
maxamillion-epel7-python-six                                                                    | 3.5 kB  00:00:00     
maxamillion-epel7-python-six/x86_64/primary_db                                                  | 4.9 kB  00:00:00     
Loading mirror speeds from cached hostfile
 * base: mirrors.advancedhosters.com
 * epel: mirror.utexas.edu
 * extras: mirror.es.its.nyu.edu
 * updates: dist1.800hosting.com
Resolving Dependencies
--> Running transaction check
---> Package python-six.noarch 0:1.3.0-4.el7 will be updated
---> Package python-six.noarch 0:1.9.0-1.el7.centos will be an update
--> Finished Dependency Resolution

Dependencies Resolved

=======================================================================================================================
 Package                Arch               Version                        Repository                              Size
=======================================================================================================================
Updating:
 python-six             noarch             1.9.0-1.el7.centos             maxamillion-atomic-reactor              29 k

Transaction Summary
=======================================================================================================================
Upgrade  1 Package

Total download size: 29 k
Is this ok [y/d/N]: y
Downloading packages:
Delta RPMs disabled because /usr/bin/applydeltarpm not installed.
python-six-1.9.0-1.el7.centos.noarch.rpm                                                        |  29 kB  00:00:00     
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Updating   : python-six-1.9.0-1.el7.centos.noarch                                                                1/2 
  Cleanup    : python-six-1.3.0-4.el7.noarch                                                                       2/2 
  Verifying  : python-six-1.9.0-1.el7.centos.noarch                                                                1/2 
  Verifying  : python-six-1.3.0-4.el7.noarch                                                                       2/2 

Updated:
  python-six.noarch 0:1.9.0-1.el7.centos                                                                               

Complete!
[root@losbs yum.repos.d]# atomic-reactor
usage: atomic-reactor [-h] [-q | -v | -V]
                      {build,create-build-image,inside-build} ...
atomic-reactor: error: too few arguments
```

**Comment by bkabrda** [comment]
I believe you're hitting this issue: https://bugzilla.redhat.com/show_bug.cgi?id=1185409
I'm afraid there's nothing we can do about this, the fix just needs to be done in RHEL. Using the newer version from your Copr repo is the best workaround ATM, I think.

**Comment by maxamillion** [comment]
I'm fine using the work around for now, but I thought I'd at least report the issue in case it wasn't on anyone's radar. Thanks for the BZ info.

**Comment by mmilata** [comment]
Closing this because the linked BZ is fixed and python-six-1.9.0-2 is available in RHEL now.
